GDJS/Runtime/inputmanager.js

```diff
@@ -41,22 +41,52 @@ gdjs.InputManager.MOUSE_RIGHT_BUTTON = 1;
 /** @constant {number} */
 gdjs.InputManager.MOUSE_MIDDLE_BUTTON = 2;
 
+/**
+ * Returns the location-modified keyCode.
+ *
+ * @private
+ */
+gdjs.InputManager.prototype.getModifiedKeyCode = function(keyCode, location) {
```

Comment:
We could call this _getLocationAwareKeyCode

"Modified" is a bit generic, LocationAware hints that the keycode is a combination of the original keycode and a location :)

Reply:
Sounds good - will do!